### DISCUSSION TITLE
IdeaVim 2.70 Unable to start a new line

### DISCUSSION THREAD

**Discussion by Alberick** [JetBrains/ideavim#746] [General]
So up until version 2.6.3 I was able to insert a new line by doing the IDE shortcut for "Start new line" which in my IDE is "shift + enter" , that would insert a new line below and it wouldn't matter on which vim mode I was.

Unfortunately with version 2.7.0, that's no longer the case, as I can only make it work if I'm in insert mode otherwise the command will just bring the caret down without actually inserting a line.

2.6.3 or previous version

https://github.com/JetBrains/ideavim/assets/5170912/57281e49-9573-401d-b57d-cbeda7e2736e

2.7.0 Behavior

https://github.com/JetBrains/ideavim/assets/5170912/d4f1c1dd-cfe8-4c17-aa25-f41efeb624e9

Is this a bug or is there a way to enable the old behavior?

**Comment by ljchen1** [comment]
Please watch/vote https://youtrack.jetbrains.com/issue/VIM-3159 for updates. Hope we will fix this issue soon.
